### PR TITLE
docs(modeler): update screenshots

### PR DESCRIPTION
**Pull request by MaxTru**
Generated with https://github.com/camunda/camunda-docs-modeler-screenshots

### CODE REVIEW

**Comment by MaxTru** [comment]
Approval is blocked by https://github.com/camunda/camunda-modeler/issues/2158

**Comment by koevskinikola** [comment]
@MaxTru thank you for submitting this update. Please ping me when the PR is unblocked.

Cheers,
Nikola

**Review by joshdaniel87 — APPROVED** [review]

**Review by joshdaniel87 — APPROVED** [review]

File: content/modeler/bpmn.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ menu:
 
 {{< img src="img/quickstart-1.png" title="New BPMN Diagram" >}}
 
-To start modeling, create a new BPMN 2.0 diagram by selecting *Create diagram > Create new BPMN diagram* in the top-level menu.
+To start modeling, create a new BPMN 2.0 diagram by selecting *Create diagram > Create new BPMN diagram (Camunda Engine)* in the top-level menu.
 
 ## Start Modeling
```
Comment (joshdaniel87) [inline content/modeler/bpmn.md:24]: []()
```suggestion
 No suggestions
```

**Review by joshdaniel87 — APPROVED** [review]

**Review by joshdaniel87 — APPROVED** [review]

**Review by joshdaniel87 — APPROVED** [review]

**Review by joshdaniel87 — APPROVED** [review]

**Review by joshdaniel87 — APPROVED** [review]

**Review by joshdaniel87 — COMMENTED** [review]
Almost

**Comment by MaxTru** [comment]
Ready for re-review, @barmac can you please have a look? Thanks

**Review by barmac — APPROVED** [review]

**Pull request merged**